galata/test/jupyterlab/inline-completer.test.ts

```diff
@@ -251,6 +251,47 @@ test.describe('Inline Completer', () => {
       await expect(ghostText).toBeHidden();
     });
 
+    test('Ghost text shows on middle of line when FIM is enabled', async ({
+      page
+    }) => {
+      //Enabling FIM
+      await page.evaluate(async () => {
+        await window.jupyterapp.commands.execute('settingeditor:open', {
+          query: 'Inline Completer'
+        });
+      });
+      await page
+        .locator(
+          '#jp-SettingsEditor-\\@jupyterlab\\/completer-extension\\:inline-completer_providers_\\@jupyterlab\\/inline-completer\\:history_autoFillInMiddle'
+        )
+        .click();
+      await page.sidebar.close();
+
+      await page.locator('#tab-key-2-1').click();
```

Comment:
Not sure what it does? Can you find a selector that does not depend on implementation details?